controllers/rcj_soccer_referee_supervisor/recorder/recorder.py

```diff
@@ -0,0 +1,70 @@
+from controller import Supervisor
+
+from pathlib import Path
+import datetime
+import time
+
+
+class VideoRecordAssistant:
+
+    def __init__(self,
+                 supervisor: Supervisor,
+                 output_path: str = "",
+                 fastforward_rate: int = 1,
+                 resolution: str = "720p"):
+
+        self.supervisor = supervisor
+        self.output_path = output_path
+        self.fastforward_rate = fastforward_rate
+        self.resolution = resolution
+
+        self.__recording = False
+
+        if not isinstance(self.supervisor, Supervisor):
+            raise TypeError("Unexpected supervisor instance")
+
+    def create_title(self):
+
+        if self.output_path == "":
+            # When output path is not specified
+            time_str = datetime.datetime.now().strftime("%Y-%m-%d-%H:%M:%S")
+            return "{}/{}.mp4".format(str(Path.home()), time_str)
+
+        return self.output_path
+
+    def get_resolution(self):
+        res_table = {"480p": (720, 480),
```

Comment:
I would put these resolutions to consts.py

Reply:
hmm, `recorder`.py is completely independent of the game, and keep it here makes more reusability.